Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/powershell:7.1.0-alpine-3.12-20201116
+FROM mcr.microsoft.com/azure-cloudshell@sha256:075eadba1d78d537fa188fe82d4bc0f6e552ab3416d60c0b43bff5fd5372535e
```

Comment:
This is strange decision on cloud-shell side not to publish versioned containers even after fact that they have versioned releases 🤔

Reply:
Yeah, it was kind of strange that they didn't even seem to have a page in dockerhub. I have this feeling that we are doing something wrong here. :D